react-carousel/my-app/src/Carousel.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  FaChevronCircleLeft,
+  FaChevronCircleRight,
+  FaCircle,
+  FaRegCircle,
+} from 'react-icons/fa';
+import { Buttons } from './Buttons';
+import { ImageDisplay } from './ImageDisplay';
+import { useEffect, useState } from 'react';
+
+export function Carousel() {
+  const [imageIndex, setImageIndex] = useState(0);
+  const images = [
```

Comment:
This should be passed in as a prop. Move this declaration to App and pass it in from there.

With the images array here, this component will _only_ display this exact set of images.